setup.py

```diff
@@ -7,10 +7,20 @@
 # licensed under the MIT license: see LICENSE.
 # -----------------------------------------------------------------------------
 
-from setuptools import setup
+from setuptools import setup, Extension
 import versioneer
 
 
+align = Extension(
+    'kevlar.alignment',
+    sources=[
+        'kevlar/alignment.c', 'third-party/ksw2/ksw2_extz.c', 'src/align.c'
```

Comment:
I initially had `kevlar/alignment.pyx` as the first source, but replaced it with `kevlar/alignment.c` after Cython did its thang. Any updates to the Cython code will require rebuilding the `kevlar/alignment.c` file.